### ISSUE TITLE
datetimee64 support for time line

### ISSUE THREAD

**Issue by dcherian** [t-makaro/animatplot#20]
Here's a screenshot of what happens when I pass a datetime64 vector.

![image](https://user-images.githubusercontent.com/2448579/48018845-cbbc1100-e0e6-11e8-9a80-72b7ca726a11.png)

**Comment by t-makaro** [comment]
I think passing in the keyword argument ```valfmt='%s'``` into the call to ```timeline_slider``` should have the same effect as what you've done in #21. Though I haven't tried it. valfmt defaults to ```'%1.2f'``` which explains what you are getting.

[This stackoverflow](https://stackoverflow.com/questions/19502506/convert-numpy-datetime64-to-string-object-in-python) might offer some insight into this. Since the portion of the datetime that people want to display (for example: jusy day/month/year vs just hour/minute) depends on the use case, I don't see a general solution for this. You may have to parse the datetime array before passing it into a ```Timeline```.

**Comment by dcherian** [comment]
Ok '%s' is a good solution. I can update my PR to do this for datetime64 automatically if you like.

 Can the slider text be moved to below the slider?

**Comment by t-makaro** [comment]
I don't think that the text can be moved below the slider, but the play/pause button can be moved by passing it a ```matplotlib.axes.Axes``` instance. Ultimately, I'd like to build a custom slider widget so that I can put proper tickmarks on the slider, but that is for the future.
